settings/src/script.js

```diff
@@ -4,12 +4,15 @@
 import { __ } from '@wordpress/i18n';
 import { reactDOM, StrictMode, useState } from '@wordpress/element';
 import { Icon, arrowLeft } from '@wordpress/icons';
+import { Spinner } from '@wordpress/components';
+import { store as coreDataStore, useEntityRecord } from '@wordpress/core-data';
```

Comment:
```suggestion
```

It doesn't look like this is used in this file.

Reply:
Yep, missed in 80b437ff76794d876b8c0b882b96005bc863995c, removed!